src/main/java/jpos/config/JposConfigException.java

```diff
@@ -28,6 +28,11 @@
  */
 public class JposConfigException extends JposException
 {
+	/**
+	 * 
```

Comment:
JavaDoc comment missing

Reply:
I removed the empty string. They had been added automatically by Eclipse when generating the serial UID to solve a compiler warning.